fairytale.md

```diff
@@ -3,10 +3,42 @@
 ------------------------------------
 
 Although many had tried, no one succeded again. 
+<<<<<<< HEAD
+
+=======
 The Artifact remained unopened, 
 and yet to be discovered was the mistery of its content. Until that day.
 The explorer and his companion stood there. Looking at it. Staring at it. Its shape, texture and smell was intriguing. 
 
+The Artifact remained unopened, and yet to be discovered was the mistery of its content. 
+Until that day, the day when everything changed, and the Fire nation attacked. 
+
+It was actually not just one day. It was many, in many different parallel universes, all happening at the same time. 
+
+The interdimensional journey began on a friday. 
+The Artifact was shining particularly bright that evening.
+Bengt, one of the local blacksmiths, approached the Artifact.
+He pulled out his axe and swung it, with all the power he had left, towards the Artifact...
+
+And Bengt was sucked into the artifact.
+
+Now is the year of 2530
+Peter dropped several drops of sauce on the Artifact when he ate chicken dumplings. He was in a panic since his father Andrey had warned him not touch it. But Peter noticed that an imperceptible crack appeared. The Artifact was like a bottle but with heavy and thick sealing wax. 
+
+Peter told his father. Andrey seemed incredible. Since this Artifact handed down by the ancestors of his family and had a history of more than 500 years, the sealing wax was hardly removed. Now no one knew what was inside and how could people open it intact. 
+
+This crack looked like an opportunity. Andrey tried to remove the wax rely on this crack. How easily the wax was removed with the secrect sauce! The father and son removed the sealing wax by spreading their sauce.
+
+The Artifact showed  true appearance -- a semitransparent bottle filled with deep brown liquid. Andrey opened the bottle and smelled. 'Oh rice vinegar!' He shouted. Rice vinegar was a kind of sauce that has a history of more than 3000 years in East Asia, but gradually disappeared in the 24rd century due to the problem of rice types.
+
+'My grandmother told me before that she had tasted the rice vinegar when she was 10. Unfortunally, she never saw this brown, awesome, fragrant liquid any longer. Rumour has it that over time, rice vinegar will become more and more mellow...' Andrey said.
+
+Peter could not wait. He have already poured out the vinegar and dipped the dumplings.
+
+Andrey did not stop him. He remembered that his grandmother had said that there were several Artifacts like this before, but only this one was left after she was 10 years old. It is said that no one can open this last bottle perfectly.  It turned out that the Artifacts stored the rice vinegar. The sealing wax cracked when met Peter's sauce, probably because of some substance in the sauce.
+
+So, he also boiled dumplings, and together with his son Peter,  tasted the dumplings with this bottle of rice vinegar that had been stored for half a century.
+>>>>>>> main
```

Comment:
This line `>>>>>> main` also needs to be removed for me to accept the merge. The content between `======` and  `>>>>>> main` represents what is in your (and my) main.

Reply:
Hi @gallenaxel 
I wonder if you can make the requested changes by **Monday 18th/Oct 12:00** at the latest, otherwise I am forced to put you on hold and go forward with another student. It's many that sent me contributions and we're a bit late now (you can see the number of pull requests skyrocketed by the deadline)
Please let me know.

Reply:
Hi! I'll make get the changes done before monday.